src/Stratis.Bitcoin.Features.Consensus/ConsensusLoop.cs

```diff
@@ -356,6 +395,12 @@ public void Stop()
                     this.Chain.SetTip(this.Tip);
                     this.logger.LogTrace("Chain reverted back to block '{0}'.", this.Tip);
 
+                    if ((blockValidationContext.Peer != null) && (blockValidationContext.BanDurationSeconds > BlockValidationContext.BanDurationNoBan))
```

Comment:
should be `!=` not `>`

Reply:
Well its a safe guard really, in this case `>` will cover all cases.